fix: update help.aiven.io link (#37)
aiven/aiven-docs: docs/products/kafka/kafka-connect/howto/gcp-pubsub-lite-sink.md

> As of version 3.0, Aiven for Apache Kafka no longer supports Confluent

Reviewer: ```suggestion
Starting with version 3.0, Aiven for Apache Kafka no longer supports Confluent
```

Reviewer: This was probably added when it was only Apache Kafka version 3.0; we now support version 3.6.

Schema Registry. For more information, read [the article describing the

> replacement, Karapace](https://aiven.io/docs/products/kafka/karapace).

Reviewer: ```suggestion
replacement, Karapace](/docs/products/kafka/karapace).
```
